package.json

```diff
@@ -75,11 +75,13 @@
     "@nx/js": "16.7.2",
     "@nx/linter": "16.7.2",
     "@nx/node": "16.7.2",
+    "@nx/playwright": "16.7.2",
     "@nx/plugin": "16.7.2",
     "@nx/web": "16.7.2",
     "@nx/workspace": "16.7.2",
     "@nxrocks/nx-spring-boot": "8.0.3",
     "@openapitools/openapi-generator-cli": "2.5.2",
+    "@playwright/test": "^1.36.0",
```

Comment:
Can you please pin the version number by running `yarn add -D @playwright/test@1.36.0`?